chat-widget/src/common/Constants.ts

```diff
@@ -182,6 +182,12 @@ export class WebChatMiddlewareConstants {
     public static readonly adaptiveCard = "AdaptiveCard";
 }
 
+export class AMSConstants {
```

Comment:
The screenshots seem to be V1's. Please test on V2 and attach evidence

Reply:
yes I used the same screenshots just for making it easy but I have two different testing env 
V1: https://josue.blob.core.windows.net/tasks/indexv1.html
V2: https://josue.blob.core.windows.net/tasks/indexv2.html
 
![image](https://user-images.githubusercontent.com/62082252/225739779-194ffb35-09d9-46b2-9e10-ff3f6cce2fc6.png)

V1 points to drop folder scritps
V2 points to dist folder scritps

I tested in both and I have the same results, that's why I just decided to use same images